src/test/java/uk/gov/hmcts/reform/sscs/service/NotificationUtilsTest.java

```diff
@@ -594,6 +594,24 @@ public void whenGettingLatestHearing_shouldReturnNullIfNoHearings() {
         assertNull(hearing);
     }
 
+    @Test
+    public void whenJointPartySubscriptionIsAdded() {
```

Comment:
the test method doesn't describe the test.
perhaps;
shouldReturnTrue_whenJointPartySubscriptionIsAdded